README.md

```diff
@@ -151,12 +188,13 @@ public class MainApplication extends Application implements ReactApplication {
   }
 }   
 ```
-
-
 ## Development
 
 See [development.md](./docs/development.md) for development docs.
 
+## Migration guide
+
+See [migration.md](./docs/migration.md) for migrating from older version of React Native libraries.
```

Comment:
See [migration.md](./docs/migration.md) for guidance on migrating from ACP React Native libraries.